client/stylesheets/_toCat.scss

```diff
@@ -2991,6 +2991,10 @@ i:hover {
     margin-top: 20px !important;
 }
 
+.managerOfferButton ul li {
```

Comment:
Merge rule `.managerOfferButton ul li` with rule on line 2990<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `managerOfferButton` should be written in lowercase with hyphens